cmake/modules/FindTensorFlow.cmake

```diff
@@ -207,34 +257,18 @@ else()
   message(STATUS "No TensorFlow-CC-LIBRARY detected")
 endif()
 
-find_path(TensorFlow_SOURCE_DIR
-        NAMES
-        tensorflow/c
-        tensorflow/cc
-        tensorflow/core
-        tensorflow/core/framework
-        tensorflow/core/platform
-        tensorflow/python
-        third_party
-        PATHS $ENV{TENSORFLOW_SOURCE_DIR})
-
-if(TensorFlow_SOURCE_DIR)
-  message(STATUS "TensorFlow-SOURCE-DIRECTORY is ${TensorFlow_SOURCE_DIR}")
-else()
-  message(STATUS "No TensorFlow source repository detected")
-endif()
 
 macro(TensorFlow_REQUIRE_C_LIBRARY)
   if(TensorFlow_C_LIBRARY)
   else()
-    message(FATAL_ERROR "Project requires libtensorflow_cc.so, please specify the path in ENV-VAR 'TENSORFLOW_BUILD_DIR'")
+    message(FATAL_ERROR "Project requires libtensorflow_cc.so, please specify the path in ENV 'export TENSORFLOW_BUILD_DIR=...' or cmake -DTensorFlow_BUILD_DIR:PATH=...")
```

Comment:
Thank you! I would like to deprecate the env-var version and just rely on `-DTensorFlow_BUILD_DIR:PATH=...`. I would alter the continuous integration test.